### ISSUE TITLE
Pilotwings 64 junk in borders

### ISSUE THREAD

**Issue by fzurita** [gonetz/GLideN64#2145]
There seems to be junk in the borders if Pilotwings 64. This was working correctly at some point.

![pilot_wings64-000](https://user-images.githubusercontent.com/11581687/69827176-3f462a00-11e4-11ea-9860-f830fd782016.png)

**Comment by gonetz** [comment]
> There seems to be junk in the borders if Pilotwings 64. This was working correctly at some point.

With PR 2

> Also, the bad horizon is back:

Could you give me a savestate? I can't reproduce it.

**Comment by fzurita** [comment]
That's the first hangglider level.

What is PR2?

**Comment by fzurita** [comment]
Just checking, these have actually been issues for a long time.

**Comment by gonetz** [comment]
> That's the first hangglider level.

It looks ok on PC.

> Just checking, these have actually been issues for a long time.

Junk  in the borders is an old issue, yes.

**Comment by fzurita** [comment]
Same with the horizon, ar least on Android. Is still present in very old builds.

**Comment by gonetz** [comment]
> Same with the horizon, at least on Android.

The ucode uses no-near-plane-clipping, which is not supported by GLES.

**Comment by standard-two-simplex** [comment]
I saw this in one of the [commits ](https://sourceforge.net/p/angrylions-stuff/code/87/)in angrylions code.

> new tests revealed the shocking truth: the VI aggresssively and regardlessly of any data alignment specificities cuts off either 7 or 8 pixels closest to h_start and h_end, so that nothing ever appears on the screen unless (h_end - h_start) is 16 pixels or greater (this removes marginal garbage graphics on the far right in Mario Tennis, Turok 2 Seeds of Evil intro and probably other games); 

Could it be that garbage appears in the original system too?

**Comment by gonetz** [comment]
> Could it be that garbage appears in the original system too?

Yes, but it is different garbage. BTW, the functionality "cut off either 7 or 8 pixels closest to h_start and h_end" is implemented in GLideN64 too.
